packages/shopify-app-remix/src/server/auth/admin/__tests__/exit-i-frame-path.test.ts

```diff
@@ -32,7 +32,7 @@ describe('authorize.admin exit iframe path', () => {
       `<script data-api-key="${config.apiKey}" src="${APP_BRIDGE_URL}"></script>`,
     );
     expect(responseText).toContain(
-      `<script>window.open("${decodeURIComponent(exitTo)}", "_top")</script>`,
+      `<script>window.open("${decodeURIComponent(exitTo)}/", "_top")</script>`,
```

Comment:
Why the addition of the "/" here? Is this because we are passing "/" instead of "" ?

Reply:
This is just because I started using an `URL` object to build this in `render-app-bridge`, and it automatically adds the `/`.